### PR TITLE
login form

### PR DESCRIPTION
**Pull request by coder-adil**
in contact form their some field like name, contact no,email id and address

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+  position: relative;
+  min-width: 1100px;
+  min-height: 550px;
+  z-index: 1000;
+}
+section .container .contactinfo {
+  position: absolute;
+  top: 40px;
+  width: 350px;
+  height: calc(100px-80px);
```
Comment (Abbas4tech) [inline index.css:35]: Can you explain how this `calc` functions work? @coder-adil

File: index.css [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+  justify-content: space-between;
+  flex-wrap: wrap;
+  padding-top: 30px;
+  width: 47%;
+  margin: 0 0 15px 0;
+  width: 100%;
+}
+section .container .mainForm .formBox .inputBox {
+  position: relative;
+  margin: 0 0 35px 0;
```
Comment (Abbas4tech) [inline index.css:131]:
```suggestion
  margin-bottom: 35px;
```

File: index.css [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+  font-weight: 500;
+}
+section .container .mainForm .formBox {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  padding-top: 30px;
+  width: 47%;
+  margin: 0 0 15px 0;
```
Comment (Abbas4tech) [inline index.css:126]:
```suggestion
  margin-bottom: 15px;
```

**Review by Abbas4tech — CHANGES_REQUESTED** [review]
Use Short hand Properties @coder-adil

File: index.css [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+}
+section .container .mainForm .formBox .inputBox.w100 {
+  width: 100%;
+}
+
+section .container .mainForm .formBox .inputBox input,
+section .container .mainForm .formBox .inputBox textarea {
+  width: 100%;
+  padding: 5px 0;
+  resize: none;
```
Comment (Abbas4tech) [inline index.css:144]: How does `resize:none` affect ??

**Review by Abbas4tech — CHANGES_REQUESTED** [review]

File: index.html [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+            <h1>Contact Information</h1>
+            <ul class="info">
+              <li>
+                <span><img src="location.png" /></span>
+                <span>Mehrun<br>
+                  Jalgaon<br>
+                  425001
+              </li>
+              <li>
+                  <span><img src="mail.png" /></span>
```
Comment (Abbas4tech) [inline index.html:23]: Add `alt` attribute!! @coder-adil

**Review by Abbas4tech — CHANGES_REQUESTED** [review]
Add `alt` attribute to all images @coder-adil

File: index.html [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+                <div class="inputBox w50">
+                  <input type="text" name="" required>
+                  <span>Last Name</span>
+                </div>
+                <div class="inputBox w50">
+                  <input type="text" name="" required>
+                  <span>Email ID</span>
+                </div>
+                <div class="inputBox w50">
+                  <input type="number" name="" required>
```
Comment (Abbas4tech) [inline index.html:54]: Remove `name=""` if it is not necessary @coder-adil

File: index.html [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+                <div class="inputBox w50">
+                    <input type="text" name="" required>
+                    <span>First Name</span>
+                </div>
+                <div class="inputBox w50">
+                  <input type="text" name="" required>
+                  <span>Last Name</span>
+                </div>
+                <div class="inputBox w50">
+                  <input type="text" name="" required>
```
Comment (Abbas4tech) [inline index.html:50]: Can you explain how `required` attribute affect this on webpage ???@coder-adil

**Review by Abbas4tech — CHANGES_REQUESTED** [review]

**Review by anask601 — COMMENTED** [review]
Group the properties @coder-adil

File: index.html [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+    <title>Document</title>
+  </head>
+  <body>
+    <section>
+      <div class="container">
+        <div class="contactinfo">
+            <h1>Contact Information</h1>
+            <ul class="info">
+              <li>
+                <span><img src="location.png" /></span>
```
Comment (anask601) [inline index.html:17]: add alt attribute to every image

File: index.css [hunk 9]
```diff
@@ -0,0 +1,187 @@
...
+  cursor: pointer;
+  align-items: flex-start;
+}
+section .container .contactinfo li span:nth-child(1) {
+  width: 30px;
+  min-width: 30px;
+}
+section .container .contactinfo li span:nth-child(1) img {
+  max-width: 100%;
+  filter: invert(1);
```
Comment (anask601) [inline index.css:68]: can u  explain what is use filter ?

File: index.css [hunk 10]
```diff
@@ -0,0 +1,187 @@
...
+}
+section {
+  position: relative;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  min-height: 100vh;
+  background-color: darkblue;
+}
+section::before {
```
Comment (mujeeb100) [inline index.css:16]: if we used _before_ properties what will happen ??
can you explain please

**Review by Tanvir04khan — COMMENTED** [review]
Group similar properties

**Pull request merged**